spotlight/evaluation.py

```diff
@@ -96,6 +96,60 @@ def sequence_mrr_score(model, test):
     return np.array(mrrs)
 
 
+def precision_recall_at_k(model, test, k):
+    """
+    Compute Precision@k and Recall@k scores. One score
+    is given for every user with interactions in the test
+    set, representing the Precision@k and Recall@k of all their
+    test items.
+
+    Parameters
+    ----------
+
+    model: fitted instance of a recommender model
+        The model to evaluate.
+    test: :class:`spotlight.interactions.Interactions`
+        Test interactions.
+    k: int or array of int,
```

Comment:
I'm not sure where I sit on making this a list:
- yes, it's more efficient;
- but it is also more complex.

Could we maybe make it so that if a scalar is passed a scalar is also returned?

Reply:
I think having two options make it more confuse. How about change it to only scalar or only list? I design it for list before, as in research, people usually plot experiment figures to see how performance changes when varying k.

Reply:
I think doing both list and scalar version is a nice compromise.

We should set `k=10` as the default value. Then
- if a list is passed, return tuple of lists of arrays
- if a scalar is passed (or the default value is kepy), we return tuple of arrays.

Reply:
OK, got it. I'll make the changes soon.

Reply:
Having said that, I would also be happy for this to be only scalar.